traffic_ops/app/lib/UI/Server.pm

```diff
@@ -912,12 +912,12 @@ sub readupdate {
 				my $rs_parents = $self->db->resultset('Server')->search( { cachegroup => $parent_cg }, { prefetch => [ 'status'] } );
```

Comment:
That makes sense.  I usually prefer to keep the filters in one place but this is simpler and cleaner.